tests/baselines/reference/user/adonis-framework.log

```diff
@@ -17,6 +17,8 @@ node_modules/adonis-framework/providers/StaticProvider.js(9,33): error TS2307: C
 node_modules/adonis-framework/providers/ViewProvider.js(9,33): error TS2307: Cannot find module 'adonis-fold'.
 node_modules/adonis-framework/src/Config/index.js(37,15): error TS2304: Cannot find name 'Mixed'.
 node_modules/adonis-framework/src/Config/index.js(39,15): error TS2304: Cannot find name 'Mixed'.
+node_modules/adonis-framework/src/Config/index.js(48,25): error TS2565: Property 'existy' is used before being assigned.
```

Comment:
This is incorrect; we shouldn't see a use-before-def error when imported from a module. Probably happens because these aren't real aliases?

Filed #27197